[100%] Testing the TCP connection...
Test project /mnt/i/DIY/CS144/sponge/build
        Start   1: t_wrapping_ints_cmp
  1/159 Test   #1: t_wrapping_ints_cmp ..............   Passed    0.02 sec
        Start   2: t_wrapping_ints_unwrap
  2/159 Test   #2: t_wrapping_ints_unwrap ...........   Passed    0.02 sec
        Start   3: t_wrapping_ints_wrap
  3/159 Test   #3: t_wrapping_ints_wrap .............   Passed    0.02 sec
        Start   4: t_recv_connect
  4/159 Test   #4: t_recv_connect ...................   Passed    0.04 sec
        Start   5: t_recv_transmit
  5/159 Test   #5: t_recv_transmit ..................   Passed    0.29 sec
        Start   6: t_recv_window
  6/159 Test   #6: t_recv_window ....................   Passed    0.28 sec
        Start   7: t_recv_reorder
  7/159 Test   #7: t_recv_reorder ...................   Passed    0.03 sec
        Start   8: t_recv_close
  8/159 Test   #8: t_recv_close .....................   Passed    0.14 sec
        Start   9: t_send_connect
  9/159 Test   #9: t_send_connect ...................   Passed    0.03 sec
        Start  10: t_send_transmit
 10/159 Test  #10: t_send_transmit ..................   Passed    0.28 sec
        Start  11: t_send_retx
 11/159 Test  #11: t_send_retx ......................   Passed    0.05 sec
        Start  12: t_send_window
 12/159 Test  #12: t_send_window ....................   Passed    0.17 sec
        Start  13: t_send_ack
 13/159 Test  #13: t_send_ack .......................   Passed    0.05 sec
        Start  14: t_send_close
 14/159 Test  #14: t_send_close .....................   Passed    0.05 sec
        Start  15: t_strm_reassem_single
 15/159 Test  #15: t_strm_reassem_single ............   Passed    0.02 sec
        Start  16: t_strm_reassem_seq
 16/159 Test  #16: t_strm_reassem_seq ...............   Passed    0.02 sec
        Start  17: t_strm_reassem_dup
 17/159 Test  #17: t_strm_reassem_dup ...............   Passed    0.04 sec
        Start  18: t_strm_reassem_holes
 18/159 Test  #18: t_strm_reassem_holes .............   Passed    0.02 sec
        Start  19: t_strm_reassem_many
 19/159 Test  #19: t_strm_reassem_many ..............   Passed    6.09 sec
        Start  20: t_strm_reassem_overlapping
 20/159 Test  #20: t_strm_reassem_overlapping .......   Passed    0.02 sec
        Start  21: t_strm_reassem_win
 21/159 Test  #21: t_strm_reassem_win ...............   Passed    7.45 sec
        Start  22: t_strm_reassem_cap
 22/159 Test  #22: t_strm_reassem_cap ...............   Passed    0.65 sec
        Start  23: t_byte_stream_construction
 23/159 Test  #23: t_byte_stream_construction .......   Passed    0.03 sec
        Start  24: t_byte_stream_one_write
 24/159 Test  #24: t_byte_stream_one_write ..........   Passed    0.02 sec
        Start  25: t_byte_stream_two_writes
 25/159 Test  #25: t_byte_stream_two_writes .........   Passed    0.03 sec
        Start  26: t_byte_stream_capacity
 26/159 Test  #26: t_byte_stream_capacity ...........   Passed    4.10 sec
        Start  27: t_byte_stream_many_writes
 27/159 Test  #27: t_byte_stream_many_writes ........   Passed    0.05 sec
        Start  28: t_webget
 28/159 Test  #28: t_webget .........................   Passed    5.38 sec
        Start  31: t_tcp_parser
 29/159 Test  #31: t_tcp_parser .....................   Passed    0.07 sec
        Start  32: t_ipv4_parser
 30/159 Test  #32: t_ipv4_parser ....................   Passed    0.07 sec
        Start  33: t_active_close
 31/159 Test  #33: t_active_close ...................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=1,ackno=1,seqno=1,)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      close

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  34: t_passive_close
 32/159 Test  #34: t_passive_close ..................***Failed    0.04 sec
Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=1,win=137) with no payload

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  36: t_ack_rst
 33/159 Test  #36: t_ack_rst ........................***Failed    0.04 sec
Test 1
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  38: t_ack_rst_win
 34/159 Test  #38: t_ack_rst_win ....................***Failed    0.04 sec
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  40: t_connect
 35/159 Test  #40: t_connect ........................***Failed    0.04 sec
Warning: Unclean shutdown of TCPConnection
Test Failure on expectation:
	exactly one segment sent with (A=1,S=0,ackno=1380966857,)
Failure message:
	The TCP produced a segment with `syn = 1`, but syn was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Action:      1ms pass
	Expectation: exactly one segment sent with (A=0,S=1,)
	Expectation: TCP in state sender=`stream started but nothing acknowledged`, receiver=`waiting for SYN: ackno is empty`, active=1, linger_after_streams_finish=1
	Action:      packet arrives: Header(flags=S,seqno=1380966856,ack=0,win=137) with no payload
	Action:      1ms pass

Note:
	test 2 failed: bad ACK for SYN

Warning: Unclean shutdown of TCPConnection
The TCP produced a segment with `syn = 1`, but syn was expected to be `0`

        Start  42: t_listen
 36/159 Test  #42: t_listen .........................***Failed    0.04 sec
Test Failure on expectation:
	Expectation: TCP in state sender=`stream started but nothing acknowledged`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1

Failure message:
	The TCP was in state `sender=`waiting for stream to begin (no SYN sent)`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`, but it was expected to be in state `sender=`stream started but nothing acknowledged`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`

List of steps that executed successfully:
	Action:      listen
	Expectation: TCP in state sender=`waiting for stream to begin (no SYN sent)`, receiver=`waiting for SYN: ackno is empty`, active=1, linger_after_streams_finish=1
	Action:      1ms pass
	Expectation: TCP in state sender=`waiting for stream to begin (no SYN sent)`, receiver=`waiting for SYN: ackno is empty`, active=1, linger_after_streams_finish=1
	Action:      packet arrives: Header(flags=S,seqno=0,ack=0,win=137) with no payload
	Action:      1ms pass
	Expectation: exactly one segment sent with (A=1,S=1,ackno=1,)

Warning: Unclean shutdown of TCPConnection
The TCP was in state `sender=`waiting for stream to begin (no SYN sent)`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`, but it was expected to be in state `sender=`stream started but nothing acknowledged`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`

        Start  43: t_winsize
 37/159 Test  #43: t_winsize ........................***Failed    0.05 sec
0Test Failure on expectation:
	Expectation: TCP in state sender=`stream ongoing`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1

Failure message:
	The TCP was in state `sender=`waiting for stream to begin (no SYN sent)`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`, but it was expected to be in state `sender=`stream ongoing`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`

List of steps that executed successfully:
	Action:      listen
	Action:      packet arrives: Header(flags=S,seqno=2697540426,ack=0,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,ackno=2697540427,win=32462,)
	Action:      packet arrives: Header(flags=A,seqno=2697540427,ack=2922477411,win=27481) with no payload
	Expectation: no (more) segments sent

Warning: Unclean shutdown of TCPConnection
The TCP was in state `sender=`waiting for stream to begin (no SYN sent)`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`, but it was expected to be in state `sender=`stream ongoing`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1`

        Start  45: t_retx
 38/159 Test  #45: t_retx ...........................***Failed    0.05 sec
  check_segment
Test Failure on expectation:
	Expectation: segment sent with (A=1,payload_size=4,data="asdf",)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=620599560,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=620599560,ack=620599561,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=620599561,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not (in check_segment called from line 36)

        Start  46: t_retx_win
 39/159 Test  #46: t_retx_win .......................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=3527744451,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=3527744451,ack=3527744452,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=3527744452,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass
	Action:      20ms pass
	Action:      write (4 bytes) [71, 77, 65, 72]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()s

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  47: t_loopback
 40/159 Test  #47: t_loopback .......................***Failed    0.04 sec
Test Failure on expectation:
	Expectation: TCP has 1745 bytes in flight

Failure message:
	The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `1745`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=400118997,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=400118997,ack=400118998,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=400118998,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=400118998,ack=400118998,win=65000) with no payload
	Action:      write (1745 bytes) [f8, 75, 2c, 8, cc, 65, c2, 7a, 48, 23, f7, 65, e4, 8f, 27, cd, ...]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `1745`

        Start  48: t_loopback_win
 41/159 Test  #48: t_loopback_win ...................***Failed    0.03 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=139594764,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=139594764,ack=139594765,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=139594765,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=139594765,ack=139594765,win=65000) with no payload
	Action:      write (2370 bytes) [0, bc, 71, 4b, f2, d0, 50, 9c, 4, 7e, 34, 8a, 45, 8d, 75, 69, ...]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  49: t_reorder
 42/159 Test  #49: t_reorder ........................***Failed    0.04 sec
Test Failure on expectation:
	segment sent with (A=1,)
Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=646864000,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=1809403596,ack=646864001,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1809403597,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=1809458959,ack=646864001,win=137) with 983 data bytes "<254>*{}<180>&Vda<246><150><21>v<158><231><229>..."

Note:
	test 1 failed: no ACK for datagram

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  50: t_address_dt
 43/159 Test  #50: t_address_dt .....................   Passed    0.04 sec
        Start  51: t_parser_dt
 44/159 Test  #51: t_parser_dt ......................   Passed    0.02 sec
        Start  52: t_socket_dt
 45/159 Test  #52: t_socket_dt ......................   Passed    0.02 sec
        Start  53: t_udp_client_send
 46/159 Test  #53: t_udp_client_send ................***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:3542... new connection from 169.254.144.1:35400.
done.
DEBUG: Outbound stream to 169.254.144.1:35400 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 27858 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  54: t_udp_server_send
 47/159 Test  #54: t_udp_server_send ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29922... new connection from 169.254.144.1:60357.
done.
DEBUG: Outbound stream to 169.254.144.1:29922 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  55: t_udp_client_recv
 48/159 Test  #55: t_udp_client_recv ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:16300... new connection from 169.254.144.1:59038.
done.
DEBUG: Outbound stream to 169.254.144.1:16300 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  56: t_udp_server_recv
 49/159 Test  #56: t_udp_server_recv ................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11492... new connection from 169.254.144.1:39602.
done.
DEBUG: Outbound stream to 169.254.144.1:39602 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 27945 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  57: t_udp_client_dupl
 50/159 Test  #57: t_udp_client_dupl ................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:13655... new connection from 169.254.144.1:37895.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 27962 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 27964 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  58: t_udp_server_dupl
 51/159 Test  #58: t_udp_server_dupl ................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:26446... new connection from 169.254.144.1:45409.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 27981 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 27986 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  59: t_ucS_1M_32k
 52/159 Test  #59: t_ucS_1M_32k .....................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:15945... new connection from 169.254.144.1:60113.
done.
DEBUG: Outbound stream to 169.254.144.1:60113 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28006 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  60: t_ucS_128K_8K
 53/159 Test  #60: t_ucS_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29294... new connection from 169.254.144.1:58600.
done.
DEBUG: Outbound stream to 169.254.144.1:58600 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28026 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  61: t_ucS_16_1
 54/159 Test  #61: t_ucS_16_1 .......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:6251... new connection from 169.254.144.1:35886.
done.
DEBUG: Outbound stream to 169.254.144.1:35886 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28046 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  62: t_ucS_32K_d
 55/159 Test  #62: t_ucS_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5728... new connection from 169.254.144.1:44186.
done.
DEBUG: Outbound stream to 169.254.144.1:44186 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28064 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  63: t_ucR_1M_32k
 56/159 Test  #63: t_ucR_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5912... new connection from 169.254.144.1:59309.
done.
DEBUG: Outbound stream to 169.254.144.1:5912 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  64: t_ucR_128K_8K
 57/159 Test  #64: t_ucR_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5181... new connection from 169.254.144.1:45980.
done.
DEBUG: Outbound stream to 169.254.144.1:5181 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  65: t_ucR_16_1
 58/159 Test  #65: t_ucR_16_1 .......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:15244... new connection from 169.254.144.1:36195.
done.
DEBUG: Outbound stream to 169.254.144.1:15244 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  66: t_ucR_32K_d
 59/159 Test  #66: t_ucR_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:4845... new connection from 169.254.144.1:33622.
done.
DEBUG: Outbound stream to 169.254.144.1:4845 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  67: t_ucD_1M_32k
 60/159 Test  #67: t_ucD_1M_32k .....................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:30165... new connection from 169.254.144.1:58771.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: /mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28216 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'Error in _rt_connect
std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 28214 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  68: t_ucD_128K_8K
 61/159 Test  #68: t_ucD_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17407... new connection from 169.254.144.1:39754.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28235 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28237 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  69: t_ucD_16_1
 62/159 Test  #69: t_ucD_16_1 .......................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11609... new connection from 169.254.144.1:55422.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28256 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28258 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  70: t_ucD_32K_d
 63/159 Test  #70: t_ucD_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18817... new connection from 169.254.144.1:46293.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28277 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  71: t_usS_1M_32k
 64/159 Test  #71: t_usS_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:3610... new connection from 169.254.144.1:36535.
done.
DEBUG: Outbound stream to 169.254.144.1:3610 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  72: t_usS_128K_8K
 65/159 Test  #72: t_usS_128K_8K ....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22560... new connection from 169.254.144.1:44508.
done.
DEBUG: Outbound stream to 169.254.144.1:22560 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  73: t_usS_16_1
 66/159 Test  #73: t_usS_16_1 .......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:2690... new connection from 169.254.144.1:60246.
done.
DEBUG: Outbound stream to 169.254.144.1:2690 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  74: t_usS_32K_d
 67/159 Test  #74: t_usS_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32195... new connection from 169.254.144.1:54768.
done.
DEBUG: Outbound stream to 169.254.144.1:32195 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  75: t_usR_1M_32k
 68/159 Test  #75: t_usR_1M_32k .....................***Failed    0.27 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25113... new connection from 169.254.144.1:52321.
done.
DEBUG: Outbound stream to 169.254.144.1:52321 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28428 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  76: t_usR_128K_8K
 69/159 Test  #76: t_usR_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:3167... new connection from 169.254.144.1:60944.
done.
DEBUG: Outbound stream to 169.254.144.1:60944 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28448 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  77: t_usR_16_1
 70/159 Test  #77: t_usR_16_1 .......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31757... new connection from 169.254.144.1:49788.
done.
DEBUG: Outbound stream to 169.254.144.1:49788 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28471 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  78: t_usR_32K_d
 71/159 Test  #78: t_usR_32K_d ......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:28807... new connection from 169.254.144.1:39173.
done.
DEBUG: Outbound stream to 169.254.144.1:39173 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28489 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  79: t_usD_1M_32k
 72/159 Test  #79: t_usD_1M_32k .....................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:8177... new connection from 169.254.144.1:34651.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28510 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  80: t_usD_128K_8K
 73/159 Test  #80: t_usD_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18721... new connection from 169.254.144.1:38693.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28529 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28531 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  81: t_usD_16_1
 74/159 Test  #81: t_usD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:13093... new connection from 169.254.144.1:44597.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28550 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28552 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  82: t_usD_32K_d
 75/159 Test  #82: t_usD_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17970... new connection from 169.254.144.1:40304.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28571 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  83: t_ucS_128K_8K_l
 76/159 Test  #83: t_ucS_128K_8K_l ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5437... new connection from 169.254.144.1:51787.
DEBUG: Outbound stream to 169.254.144.1:51787 finished (0 bytes still in flight).

        Start  84: t_ucS_128K_8K_L
 77/159 Test  #84: t_ucS_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5982... new connection from 169.254.144.1:55905.
done.
DEBUG: Outbound stream to 169.254.144.1:55905 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28624 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  85: t_ucS_128K_8K_lL
 78/159 Test  #85: t_ucS_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:7527... new connection from 169.254.144.1:55766.
done.
DEBUG: Outbound stream to 169.254.144.1:55766 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28644 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  86: t_ucR_128K_8K_l
 79/159 Test  #86: t_ucR_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18721... new connection from 169.254.144.1:50299.
done.
DEBUG: Outbound stream to 169.254.144.1:18721 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  87: t_ucR_128K_8K_L
 80/159 Test  #87: t_ucR_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31689... new connection from 169.254.144.1:55478.
done.
DEBUG: Outbound stream to 169.254.144.1:31689 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  88: t_ucR_128K_8K_lL
 81/159 Test  #88: t_ucR_128K_8K_lL .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32618... 

        Start  89: t_ucD_128K_8K_l
 82/159 Test  #89: t_ucD_128K_8K_l ..................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:30394... new connection from 169.254.144.1:51417.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28776 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28781 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  90: t_ucD_128K_8K_L
 83/159 Test  #90: t_ucD_128K_8K_L ..................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:33129... new connection from 169.254.144.1:33841.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28800 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 28802 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  91: t_ucD_128K_8K_lL
 84/159 Test  #91: t_ucD_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11534... 

        Start  92: t_usS_128K_8K_l
 85/159 Test  #92: t_usS_128K_8K_l ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11828... new connection from 169.254.144.1:41684.
done.
DEBUG: Outbound stream to 169.254.144.1:11828 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  93: t_usS_128K_8K_L
 86/159 Test  #93: t_usS_128K_8K_L ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22393... new connection from 169.254.144.1:39070.
done.
DEBUG: Outbound stream to 169.254.144.1:22393 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  94: t_usS_128K_8K_lL
 87/159 Test  #94: t_usS_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:2539... new connection from 169.254.144.1:51409.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  95: t_usR_128K_8K_l
 88/159 Test  #95: t_usR_128K_8K_l ..................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:16822... new connection from 169.254.144.1:35945.
done.
DEBUG: Outbound stream to 169.254.144.1:35945 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29038 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  96: t_usR_128K_8K_L
 89/159 Test  #96: t_usR_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31893... new connection from 169.254.144.1:37279.
done.
DEBUG: Outbound stream to 169.254.144.1:37279 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29058 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  97: t_usR_128K_8K_lL
 90/159 Test  #97: t_usR_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21240... new connection from 169.254.144.1:44355.
done.
DEBUG: Outbound stream to 169.254.144.1:44355 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29078 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  98: t_usD_128K_8K_l
 91/159 Test  #98: t_usD_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:27797... new connection from 169.254.144.1:38000.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29099 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  99: t_usD_128K_8K_L
 92/159 Test  #99: t_usD_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23889... new connection from 169.254.144.1:50968.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29120 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 100: t_usD_128K_8K_lL
 93/159 Test #100: t_usD_128K_8K_lL .................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14762... new connection from 169.254.144.1:46901.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29139 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29141 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 101: t_ipv4_client_send
 94/159 Test #101: t_ipv4_client_send ...............***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:9688... new connection from 169.254.144.9:36356.
DEBUG: Outbound stream to 169.254.144.9:36356 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:36356 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29159 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 102: t_ipv4_server_send
 95/159 Test #102: t_ipv4_server_send ...............***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:6045... new connection from 169.254.145.9:63782.
done.
DEBUG: Outbound stream to 169.254.144.9:6045 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 103: t_ipv4_client_recv
 96/159 Test #103: t_ipv4_client_recv ...............***Timeout  10.02 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:28210... new connection from 169.254.144.9:43033.
done.
DEBUG: Outbound stream to 169.254.145.9:28210 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 104: t_ipv4_server_recv
 97/159 Test #104: t_ipv4_server_recv ...............***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:33265... new connection from 169.254.145.9:9312.
DEBUG: Outbound stream to 169.254.145.9:9312 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.145.9:9312 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29247 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 105: t_ipv4_client_dupl
 98/159 Test #105: t_ipv4_client_dupl ...............***Failed    0.21 sec
DEBUG: Connecting to DEBUG: Listening for incoming connection... 169.254.145.9:32667... new connection from 169.254.144.9:23616.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29264 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29266 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 106: t_ipv4_server_dupl
 99/159 Test #106: t_ipv4_server_dupl ...............***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15321... new connection from 169.254.145.9:34782.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 107: t_icS_1M_32k
100/159 Test #107: t_icS_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:10667... new connection from 169.254.144.9:34204.
DEBUG: Outbound stream to 169.254.144.9:34204 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:34204 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29322 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 108: t_icS_128K_8K
101/159 Test #108: t_icS_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:12960... new connection from 169.254.144.9:7893.
DEBUG: Outbound stream to 169.254.144.9:7893 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:7893 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29342 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 109: t_icS_16_1
102/159 Test #109: t_icS_16_1 .......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:14007... new connection from 169.254.144.9:45575.
DEBUG: Outbound stream to 169.254.144.9:45575 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:45575 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29362 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 110: t_icS_32K_d
103/159 Test #110: t_icS_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:2067... new connection from 169.254.144.9:12256.
DEBUG: Outbound stream to 169.254.144.9:12256 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:12256 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29380 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 111: t_icR_1M_32k
104/159 Test #111: t_icR_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:8228... new connection from 169.254.144.9:63051.
done.
DEBUG: Outbound stream to 169.254.145.9:8228 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 112: t_icR_128K_8K
105/159 Test #112: t_icR_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:31512... new connection from 169.254.144.9:475.
done.
DEBUG: Outbound stream to 169.254.145.9:31512 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 113: t_icR_16_1
106/159 Test #113: t_icR_16_1 .......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:20654... new connection from 169.254.144.9:20018.
done.
DEBUG: Outbound stream to 169.254.145.9:20654 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 114: t_icR_32K_d
107/159 Test #114: t_icR_32K_d ......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:21083... new connection from 169.254.144.9:28773.
done.
DEBUG: Outbound stream to 169.254.145.9:21083 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 115: t_icD_1M_32k
108/159 Test #115: t_icD_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:3789... new connection from 169.254.144.9:60954.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29539 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29541 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 116: t_icD_128K_8K
109/159 Test #116: t_icD_128K_8K ....................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:27429... new connection from 169.254.144.9:6625.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29560 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29562 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 117: t_icD_16_1
110/159 Test #117: t_icD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:13605... new connection from 169.254.144.9:22921.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29581 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29583 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 118: t_icD_32K_d
111/159 Test #118: t_icD_32K_d ......................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:29691... new connection from 169.254.144.9:44511.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29600 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29602 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 119: t_isS_1M_32k
112/159 Test #119: t_isS_1M_32k .....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:30919... new connection from 169.254.145.9:969.
done.
DEBUG: Outbound stream to 169.254.144.9:30919 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 120: t_isS_128K_8K
113/159 Test #120: t_isS_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:13892... new connection from 169.254.145.9:541.
done.
DEBUG: Outbound stream to 169.254.144.9:13892 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 121: t_isS_16_1
114/159 Test #121: t_isS_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:26657... new connection from 169.254.145.9:30246.
done.
DEBUG: Outbound stream to 169.254.144.9:26657 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 122: t_isS_32K_d
115/159 Test #122: t_isS_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15421... new connection from 169.254.145.9:51109.
done.
DEBUG: Outbound stream to 169.254.144.9:15421 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 123: t_isR_1M_32k
116/159 Test #123: t_isR_1M_32k .....................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:7652... new connection from 169.254.145.9:6953.
DEBUG: Outbound stream to 169.254.145.9:6953 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.145.9:6953 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29759 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 124: t_isR_128K_8K
117/159 Test #124: t_isR_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:3234... new connection from 169.254.145.9:12098.
DEBUG: Outbound stream to 169.254.145.9:12098 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.145.9:12098 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29782 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 125: t_isR_16_1
118/159 Test #125: t_isR_16_1 .......................***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:24387... new connection from 169.254.145.9:58541.
DEBUG: Outbound stream to 169.254.145.9:58541 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.145.9:58541 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29802 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 126: t_isR_32K_d
119/159 Test #126: t_isR_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:3471... new connection from 169.254.145.9:23058.
DEBUG: Outbound stream to done.
169.254.145.9:23058 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.145.9:23058 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29820 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 127: t_isD_1M_32k
120/159 Test #127: t_isD_1M_32k .....................***Failed    0.34 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:7788... new connection from 169.254.145.9:38861.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29839 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29841 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 128: t_isD_128K_8K
121/159 Test #128: t_isD_128K_8K ....................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:26968... new connection from 169.254.145.9:20703.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29862 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 129: t_isD_16_1
122/159 Test #129: t_isD_16_1 .......................***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:31466... new connection from 169.254.145.9:50907.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29884 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 130: t_isD_32K_d
123/159 Test #130: t_isD_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:9161... new connection from 169.254.145.9:23835.
done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29903 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 131: t_icS_128K_8K_l
124/159 Test #131: t_icS_128K_8K_l ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:10982... new connection from 169.254.144.9:63028.
DEBUG: Outbound stream to 169.254.144.9:63028 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:63028 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29923 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 132: t_icS_128K_8K_L
125/159 Test #132: t_icS_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:27017... new connection from 169.254.144.9:63955.
DEBUG: Outbound stream to 169.254.144.9:63955 finished (0 bytes still in flight).
done.
DEBUG: Outbound stream to 169.254.144.9:63955 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 29944 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 133: t_icS_128K_8K_lL
CMakeFiles/check_lab4.dir/build.make:57: recipe for target 'CMakeFiles/check_lab4' failed
make[3]: *** [CMakeFiles/check_lab4] Interrupt
CMakeFiles/Makefile2:131: recipe for target 'CMakeFiles/check_lab4.dir/all' failed
make[2]: *** [CMakeFiles/check_lab4.dir/all] Interrupt
CMakeFiles/Makefile2:138: recipe for target 'CMakeFiles/check_lab4.dir/rule' failed
make[1]: *** [CMakeFiles/check_lab4.dir/rule] Interrupt
Makefile:155: recipe for target 'check_lab4' failed
make: *** [check_lab4] Interrupt
